src/lambda/vue/samInvoke.ts

```diff
@@ -0,0 +1,501 @@
+import { getLogger } from '../../shared/logger'
+import { CompositeResourceFetcher } from '../../shared/resourcefetcher/compositeResourceFetcher'
+import { FileResourceFetcher } from '../../shared/resourcefetcher/fileResourceFetcher'
+import { HttpResourceFetcher } from '../../shared/resourcefetcher/httpResourceFetcher'
+import { ResourceFetcher } from '../../shared/resourcefetcher/resourcefetcher'
+import {
+    AwsSamDebuggerConfiguration,
+    isCodeTargetProperties,
+    isTemplateTargetProperties,
+} from '../../shared/sam/debugger/awsSamDebugConfiguration'
+import { DefaultAwsSamDebugConfigurationValidator } from '../../shared/sam/debugger/awsSamDebugConfigurationValidator'
+import { SamDebugConfigProvider } from '../../shared/sam/debugger/awsSamDebugger'
+import * as input from '../../shared/ui/input'
+import * as picker from '../../shared/ui/picker'
+import { addCodiconToString } from '../../shared/utilities/textUtilities'
+import { createVueWebview } from '../../webviews/main'
+import { sampleRequestManifestPath, sampleRequestPath } from '../constants'
+import { tryGetAbsolutePath } from '../../shared/utilities/workspaceUtils'
+import { CloudFormation } from '../../shared/cloudformation/cloudformation'
+import { openLaunchJsonFile } from '../../shared/sam/debugger/commands/addSamDebugConfiguration'
+import { recordSamOpenConfigUi } from '../../shared/telemetry/telemetry.gen'
+
+const localize = nls.loadMessageBundle()
+
+export function registerSamInvokeVueCommand(context: ExtContext): vscode.Disposable {
+    return vscode.commands.registerCommand(
+        'aws.lambda.invokeForm',
+        async (launchConfig?: AwsSamDebuggerConfiguration) => {
+            await createVueWebview<SamInvokerRequest, SamInvokerResponse>({
+                id: 'createLambda',
+                name: localize('AWS.command.lambda.editor.title', 'SAM Debug Configuration Editor'),
+                webviewJs: 'samInvokeVue.js',
+                onDidReceiveMessageFunction: async (message, postMessageFn, destroyWebviewFn) =>
+                    handleFrontendToBackendMessage(message, postMessageFn, destroyWebviewFn, context),
+                context: context.extensionContext,
+                cssFiles: ['samInvokeForm.css'],
+                initialCalls: launchConfig
+                    ? [
+                          {
+                              command: 'loadSamLaunchConfig',
+                              data: {
+                                  launchConfig: launchConfig,
+                              },
+                          },
+                      ]
+                    : undefined,
+            })
+
+            recordSamOpenConfigUi()
+        }
+    )
+}
+
+export interface SamInvokeVueState {
+    launchConfig: MorePermissiveAwsSamDebuggerConfiguration
+    payload: { value: string; errorMsg: string }
+}
+
+export interface MorePermissiveAwsSamDebuggerConfiguration extends AwsSamDebuggerConfiguration {
```

Comment:
using a common prefix helps discoverability. We can also avoid some verbosity:
```suggestion
export interface AwsSamDebuggerConfigurationLoose extends AwsSamDebuggerConfiguration {
```